scripts/postgres_setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Thing shim around the normal Docker postgres entrypoint that allows us to run
+# non-application migrations. Things like DB and user creations that would be
+# done by cloud-automation tasks in a normal env.
+
+set -e
+
+# Initialize the DB, but don't allow outside connections yet.
+docker-entrypoint.sh postgres -c listen_addresses='127.0.0.1' &
+# Wait until the server is out of initialization mode and online.
+while ! psql -U postgres -h localhost -c 'SELECT 1;' 2>/dev/null; do sleep 1; done
+# Stop the server.
+gosu postgres pg_ctl stop
+
+echo "[postgres] run migrations"
+
+# Run migrations/scripts that should run on every start. This is handy for data
+# we want to backfill or otherwise migrate for users.
+gosu postgres bash -c "(
+  source /usr/local/bin/docker-entrypoint.sh
```

Comment:
Official maintainers have [considered](https://github.com/docker-library/postgres/issues/821) that users like us may want this sort of behavior where we can run migrations or other scripts against Postgres and [have done some work](https://github.com/docker-library/postgres/pull/496#issue-214478296) that allows us to hook into their entrypoint and create our own for this sort of behavior.